options/locale/locale_en-US.ini

```diff
@@ -1312,6 +1312,8 @@ editor.file_editing_no_longer_exists = The file being edited, "%s", no longer ex
 editor.file_deleting_no_longer_exists = The file being deleted, "%s", no longer exists in this repository.
 editor.file_changed_while_editing = The file contents have changed since you started editing. <a target="_blank" rel="noopener noreferrer" href="%s">Click here</a> to see them or <strong>Commit Changes again</strong> to overwrite them.
 editor.file_already_exists = A file named "%s" already exists in this repository.
+editor.commit_id_not_matching = The Commit ID does not match the ID when you began editing.  Commit into a patch branch and then merge.
```

Comment:
Wouldn't "Commit hash" be a more accurate name in git terminology?

Reply:
I wanted the message to match the key.

Reply:
Hmm I see the term "Commit ID" is deeply rooted used in the backend, so I guess it's ok

Reply:
Yeah, there's a mess all over as far as consistency goes in the locale file.  I just wanted to get to the bottom of which case was being triggered when I got the error, because mine was definitely not  the file changing.